File: app/src/main/java/com/parishod/watomatic/NotificationWear.java
package com.parishod.watomatic;

import android.app.Notification;
import android.app.PendingIntent;
import android.os.Bundle;

import androidx.core.app.RemoteInput;

import java.util.ArrayList;
import java.util.UUID;

public class NotificationWear {
    public String id = UUID.randomUUID().toString();
    public String packageName = "";
    public PendingIntent pendingIntent;
    public ArrayList<RemoteInput> remoteInputs = new ArrayList<>();
    public ArrayList<Notification> pages = new ArrayList<>();
    public Bundle bundle;
    public String tag;
}

File: app/src/main/java/com/parishod/watomatic/NotificationWear.kt
package com.parishod.watomatic

import android.app.PendingIntent
import android.os.Bundle
import androidx.core.app.RemoteInput

data class NotificationWear (
        val packageName: String,
        val pendingIntent: PendingIntent?,
        val remoteInputs: List<RemoteInput>,
        val bundle: Bundle?,
        val tag: String?, // Tag can be null for some notifications
        val id: String
)